src/components/@shared/ButtonBuy/index.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { FormEvent, ReactElement } from 'react'
-import Button from '../atoms/Button'
+import Button from '../atoms/Button/index'
```

Comment:
do we really have to do this with the `/index` at the end? Cause then we have to touch almost every file in the codebase